docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,669 @@
+
+[source,shell]
+----
+mvn clean package -DskipTests=true
+----
+For information about developing the REST Service using JAX-RS and Helidon API, see https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/integrating-oracle-weblogic-server-rest-services-helidon.html#GUID-AB8F2DC5-840B-40E7-89B4-7FDAD8035C88[Integrating WebLogic REST Services with Helidon 2.x].
+
+After you have compiled, start the Helidon server using the following command:
+
+[source,shell]
+----
+java -jar ./target/helidon-test.jar
+----
+Start the Helidon server to access the REST service running in Oracle WebLogic Server by using the Helidon URL. The URL will  use the name and port number of the host where the Helidon microservice application is running.
+
+[source,java]
+----
+http://<HELIDON_HOST>:<HELIDON_PORT>/client/<handler>
+----
+To test the application, send a request from the Helidon endpoint using one of the following:
+
+* A Java client
+* A browser
+* Postman 
+
+You should receive a response from the REST service deployed in Oracle WebLogic Server. For example, you should receive the following response when you initiate the `getHelidonClientToWlsRestText` handler:
+
+[source,java]
+----
+Hi there!
+----
+
+=== References [[REST-Services-References]]
+https://helidon.io/docs/v2/#/mp/guides/02_quickstart[Helidon MP Quickstart]
+
+== JMS Integration [[JMS-Integration]]
+
+- <<JMS-Integration-Overview, Overview>>
+- <<JMS-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<JMS-Integration-Usage, Usage>>
+- <<JMS-Integration-References, References>>
+
+=== Overview [[JMS-Integration-Overview]]
+
+The integration of the Oracle WebLogic Server Java Message Service (JMS) with Helidon enables Oracle WebLogic Server applications and Helidon microservices to communicate through messaging, in one or both directions.
+
+=== Maven Coordinates [[JMS-Integration-Maven-Coordinates]]
+
+To enable Oracle WebLogic Server JMS  with Helidon, add the following dependencies to your project’s `pom.xml` file:
+
+*Dependency for Reactive Messaging*
+
+[source,xml] 
+----
+<dependency>
+   <groupId>io.helidon.microprofile.messaging</groupId>
+   <artifactId>helidon-microprofile-messaging</artifactId>
+</dependency>
+----
+*Dependency for JMS connector*
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.messaging.jms</groupId>
+    <artifactId>helidon-messaging-jms</artifactId>
+</dependency>
+----
+*Dependency for Messaging Health Metrics*
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.messaging</groupId>
+    <artifactId>helidon-microprofile-messaging-health</artifactId>
+</dependency>
+----
+
+*Dependencies for the WLS thin client JAR file*
+
+Download the Jakarta thin client JAR (`wlthint3client.jakarta`) file from https://edelivery.oracle.com/osdc/faces/Home.jspx[Oracle Software Delivery Cloud], and add it as part of the Maven dependencies:
```

Comment:
This client is not jakarta it is called wlthint3client and can not be downloaded from edelivery.  The wlthint3client is generated when WebLogic Server is installed and  It is located in the WL_HOME\server\lib directory of the WebLogic Server installation.

Reply:
Hi @Geetha-Savithriamma 
Helidon2 is supported only with the wlthint3client jar.  Jakarta thin client JAR is only for Helidon3.

Hi @mriccell 
We can use the wlthint3client jar from the path WL_HOME\server\lib. But as per the latest updates from the WebLogic 
Development team, the multi-version wlthint3client jar also will be delivered separately along with Jakarta thin client jar.  Could you please clarify this?